codalab/model/tables.py

```diff
@@ -228,3 +228,18 @@
   Index('user_user_name_index', 'user_name'),
   sqlite_autoincrement=True,
 )
+
+# Store information about users' questions or feedback.
+chat = Table(
+  'chat',
+  db_metadata,
+  Column('id', Integer, primary_key=True, nullable=False),
+  Column('time', DateTime, nullable=False),  # When did the user send this query?
+  Column('date', String(63), nullable=False),  # (e.g., 2015-09-11)
+  Column('user_id', String(63), nullable=True),  # Who did it?
+  Column('user_name', String(63), nullable=True),  # Who did it?
+  Column('chat', String(63), nullable=False),  # What's the content of the chat?
+  Column('is_answered', Boolean), # Has this chat been answered?
```

Comment:
Sometimes we only wanna query chats that are un-answered. I think making it a column will be easier?